tabbycat/draw/generator/random.py

```diff
@@ -2,7 +2,7 @@
 
 import random
 
-from django.utils.translation import gettext as _
+from django.utils.translation import gettext as gettext
```

Comment:
The `as` keyword "renames" the import. You're renaming a function to itself, so it isn't necessary to have an `as`.

Reply:
Oops!
I admit, I used find and replace to do that.